scripts/seed_datastore.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+#
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Fetches features from the staging or live dashboard site, and inserts them into the development
+datastore.
+
+This script can also set a user as admin, to facilitate testing those parts of the site.
+"""
+
+from datetime import datetime
+from google.cloud import ndb
+from urllib.parse import urljoin
+import argparse
+import logging
+import os
+import pprint
+import requests
+import sys
+
+sys.path = [os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+            ] + sys.path
+os.environ['DATASTORE_EMULATOR_HOST'] = 'localhost:15606'
```

Comment:
Use setdefault for these four environment variables in case the caller of this script wants to set their own values.

Example: `os.environ.setdefault('DATASTORE_EMULATOR_HOST', 'localhost:15606')`

Reply:
Done.